src/widgetresize/resizerstate.js

```diff
@@ -146,6 +141,23 @@ export default class ResizeState {
 		this.proposedHandleHostWidth = newSize.handleHostWidth;
 		this.proposedHandleHostHeight = newSize.handleHostHeight;
 	}
+
+	/**
+	 * Calculates a relative width of a `domResizeHost` compared to it's parent in percents.
+	 *
+	 * @protected
+	 * @param {HTMLElement} domResizeHost
+	 * @returns {Number}
+	 */
+	_getParentPercentage( domResizeHost ) {
```

Comment:
```suggestion
	_calculateHostPercentageWidth( domResizeHost ) {
```

Reply:
Plus, this function has nothing to do with this class – you can see that it does not access `this` even once. Hence, it's just a helper.